{"cvrfdoc":{"document_title":"Red Hat Security Advisory: python security update","document_distribution":"Copyright Â© 2021 Red Hat, Inc. All rights reserved.","document_references":{"reference":[{"description":"https://access.redhat.com/errata/RHSA-2005:108","type":"Self","url":"https://access.redhat.com/errata/RHSA-2005:108"},{"description":"http://www.python.org/security/PSF-2005-001/","type":"External","url":"http://www.python.org/security/PSF-2005-001/"}]},"aggregate_severity":"Important","document_tracking":{"initial_release_date":"2005-02-15T09:33:00Z","identification":{"id":"RHSA-2005:108"},"revision_history":{"revision":{"date":"2005-02-15T00:00:00Z","number":"1","description":"Current version"}},"generator":{"date":"2021-08-23T04:27:00Z","engine":"Red Hat SDEngine 3.0.0"},"current_release_date":"2005-02-15T00:00:00Z","version":"1","status":"Final"},"product_tree":{"relationship":[{"relates_to_product_reference":"4AS","product_reference":"python-0:2.3.4-14.1","full_product_name":{"product_id":"4AS:python-0:2.3.4-14.1","cpe":"cpe:/o:redhat:enterprise_linux:4::as","product_name":"python-0:2.3.4-14.1 as a component of Red Hat Enterprise Linux AS version 4"},"relation_type":"Default Component Of"},{"relates_to_product_reference":"4Desktop","product_reference":"python-0:2.3.4-14.1","full_product_name":{"product_id":"4Desktop:python-0:2.3.4-14.1","cpe":"cpe:/o:redhat:enterprise_linux:4::desktop","product_name":"python-0:2.3.4-14.1 as a component of Red Hat Enterprise Linux Desktop version 4"},"relation_type":"Default Component Of"},{"relates_to_product_reference":"4ES","product_reference":"python-0:2.3.4-14.1","full_product_name":{"product_id":"4ES:python-0:2.3.4-14.1","cpe":"cpe:/o:redhat:enterprise_linux:4::es","product_name":"python-0:2.3.4-14.1 as a component of Red Hat Enterprise Linux ES version 4"},"relation_type":"Default Component Of"},{"relates_to_product_reference":"4WS","product_reference":"python-0:2.3.4-14.1","full_product_name":{"product_id":"4WS:python-0:2.3.4-14.1","cpe":"cpe:/o:redhat:enterprise_linux:4::ws","product_name":"python-0:2.3.4-14.1 as a component of Red Hat Enterprise Linux WS version 4"},"relation_type":"Default Component Of"}],"branch":[{"name":"Red Hat Enterprise Linux","type":"Product Family","branch":[{"name":"Red Hat Enterprise Linux AS version 4","full_product_name":{"product_id":"4AS","cpe":"cpe:/o:redhat:enterprise_linux:4::as","product_name":"Red Hat Enterprise Linux AS version 4"},"type":"Product Name"},{"name":"Red Hat Enterprise Linux Desktop version 4","full_product_name":{"product_id":"4Desktop","cpe":"cpe:/o:redhat:enterprise_linux:4::desktop","product_name":"Red Hat Enterprise Linux Desktop version 4"},"type":"Product Name"},{"name":"Red Hat Enterprise Linux ES version 4","full_product_name":{"product_id":"4ES","cpe":"cpe:/o:redhat:enterprise_linux:4::es","product_name":"Red Hat Enterprise Linux ES version 4"},"type":"Product Name"},{"name":"Red Hat Enterprise Linux WS version 4","full_product_name":{"product_id":"4WS","cpe":"cpe:/o:redhat:enterprise_linux:4::ws","product_name":"Red Hat Enterprise Linux WS version 4"},"type":"Product Name"}]},{"name":"python-0:2.3.4-14.1","full_product_name":{"product_id":"python-0:2.3.4-14.1","cpe":"cpe:/o:redhat:enterprise_linux:4::as","product_name":"python-2.3.4-14.1.src.rpm"},"type":"Product Version"}]},"document_publisher":{"issuing_authority":"Red Hat Product Security","contact_details":"secalert@redhat.com","type":"Vendor"},"vulnerability":[{"notes":{"note":"security flaw"},"cve":"CVE-2005-0089","references":{"reference":[{"description":"CVE-2005-0089","url":"https://access.redhat.com/security/cve/CVE-2005-0089"},{"description":"bz#1617451: CVE-2005-0089 security flaw","url":"https://bugzilla.redhat.com/show_bug.cgi?id=1617451"}]},"release_date":"2005-02-03T00:00:00Z","involvements":{"involvement":{"party":"Vendor","status":"Completed"}},"product_statuses":{"status":{"product_id":["4AS:python-0:2.3.4-14.1","4Desktop:python-0:2.3.4-14.1","4ES:python-0:2.3.4-14.1","4WS:python-0:2.3.4-14.1"],"type":"Fixed"}},"remediations":{"remediation":{"description":"Before applying this update, make sure that all previously-released\nerrata relevant to your system have been applied.  Use Red Hat\nNetwork to download and update your packages.  To launch the Red Hat\nUpdate Agent, use the following command:\nup2date\nFor information on how to install packages manually, refer to the\nfollowing Web page for the System Administration or Customization\nguide specific to your system:\nhttp://www.redhat.com/docs/manuals/enterprise/","type":"Vendor Fix","url":"https://access.redhat.com/errata/RHSA-2005:108"}},"threats":{"threat":{"description":"Important","type":"Impact"}},"discovery_date":"2005-01-30T00:00:00Z","ordinal":"1"}],"document_notes":{"note":["Updated Python packages that fix several security issues are now available\nfor Red Hat Enterprise Linux 4.\nThis update has been rated as having important security impact by the Red\nHat Security Response Team","Python is an interpreted, interactive, object-oriented programming language.\nAn object traversal bug was found in the Python SimpleXMLRPCServer.  This\nbug could allow a remote untrusted user to do unrestricted object traversal\nand allow them to access or change function internals using the im_* and\nfunc_* attributes.  The Common Vulnerabilities and Exposures project\n(cve.mitre.org) has assigned the name CAN-2005-0089 to this issue.\nUsers of Python are advised to upgrade to these updated packages, which\ncontain backported patches to correct these issues.","This content is licensed under the Creative Commons Attribution 4.0 International License (https://creativecommons.org/licenses/by/4.0/). If you distribute this content, or a modified version of it, you must provide attribution to Red Hat Inc. and provide a link to the original."]},"document_type":"Security Advisory"}}